assets/js/modules/thank-with-google/datastore/publications.js

```diff
@@ -155,6 +167,42 @@ const baseSelectors = {
 			publications[ 0 ]
 		);
 	} ),
+
+	/**
+	 * Gets the link to create new publication.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return {string} Create publication URL.
+	 */
+	getServiceCreatePublicationURL: createRegistrySelector(
+		( select ) => () => {
+			const homeURL = select( CORE_SITE ).getHomeURL();
+
+			const url = addQueryArgs( 'https://publishercenter.google.com/', {
+				sk_url: encodeURIComponent( homeURL ),
```

Comment:
This `encodeURIComponent` call needs to be removed, since `addQueryArgs` already handles encoding of parameters. Therefore now the parameter is double encoded which is incorrect.